### PR TITLE
Update SonarQube configuration

### PR DESCRIPTION
**Pull request by EMazzaglia**
## Pull request type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?
Sonar configuration was included on the github pipeline.
Test folder is included on the sonar scanner. 

Issue Link: N/A

## What is the new behavior?
Sonar properties file is now versioned.
Test folder is excluded from the sonar scanner.

## Other information
We removed the sonar-scanner config from the pipeline. 

## Preview
N/A

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#----- Project Configuration
+sonar.sources=src
+sonar.projectBaseDir=.
+sonar.projectVersion=1.0.0
+sonar.javascript.lcov.reportPaths=./coverage/lcov.info
+sonar.testExecutionReportPaths=__tests__/test-report.xml
+sonar.tests=__tests__
+sonar.test.inclusions=**/*.spec.ts
+sonar.coverage.exclusions=__tests__/**/*
+sonar.verbose=false
```
Comment (zk182) [inline sonar-project.properties:10]: Maybe for productive environments we might want to be verbose

**Review by zk182 — APPROVED** [review]
LGTM

File: sonar-project.properties (hunk 1 above)
Comment (EMazzaglia) [reply sonar-project.properties:10]: Yes, I thought about that, but since this is a base this should be fine for now.

**Review by nelegatti — APPROVED** [review]
Looks good, nice work 🚀

**Review by agustinpurita — APPROVED** [review]
LGTM!

**Pull request merged**